spec/onchain.tex

```diff
@@ -414,19 +469,60 @@ \subsection{Close Transaction}\label{sec:close-tx}
 		\[
 			\eta' = \eta
 		\]
-		\item $\mathsf{Unused}$: Closing snapshot refers to current state version $v$ and any UTxO to decommit need to be present in the closed state too.
+    \red{
+		  \item $\mathsf{UnusedInc}$: Closing snapshot refers to current state version $v$ and any UTxO to increment must be $\bot$ in the closed state.
+		  \[
+		  	\eta_\alpha = \eta_\Delta' = \bot
```

Comment:
This is wrong: $\eta_\alpha$ might be $\neq \bot$!

```suggestion
		  	\eta_\Delta' = \bot
```

Reply:
How come? In the `UnusedInc` we provide $\eta_\alpha$ in the redeemer but it should be $\bot$ in the closed state (which is $\eta_\Delta$ here)

Reply:
As it's a snapshot that approves an increment, it naturally would include a UTxO to commit. Consequently $\eta_\alpha$ of the snapshot signature is $\eta_\alpha \neq \bot$

Reply:
ok we are on the same page, only $\eta_\Delta$ is $\bot$ in the closed state